packages/esm-patient-list-app/src/patientListData/setupMockState.ts

```diff
@@ -0,0 +1,32 @@
+import { addPatientToPatientList, createPatientList, newUuid, updatePatientListDetails } from './mock';
+import { PATIENT_LIST_TYPE } from './types';
+
+const setup = async () => {
```

Comment:
Any reason for using `async` / `await` in here, but not in the other cases (e.g., in `addPatientToPatientList` or `deletePatientFromPatientList` etc.)?

Reply:
Otherwise, it sounds like `setup` is a good candidate for a `function`.

Reply:
`addPatientToPatientList` etc were one-line arrow functions before, so there was no need for async/await. Async/await would be cleaner but again, this code is to be deleted.

Reply:
Alright - just keep consistency please.